content/posts/2021-11-18-relicensing-update.md

```diff
@@ -0,0 +1,328 @@
+---
+author: Kristof Beyls
+date: "2021-11-18"
+tags: ["relicensing", "foundation"]
+title: LLVM relicensing update & call for help
+---
+
+In this blog post, I'd like to summarize the main points I talked about in the
+relicensing update presentation at the 2021 LLVM developers meeting.
+
+The very short summary is that we are currently in the long tail phase of
+collecting relicensing agreements of past contributors. We already at the time
+of this writing have more than 94% of older code relicensed.  We hope to crowd
+source getting through the long tail to get as close to 100% as possible.
+
+# Call for help
+
+You can help by looking through [the list of remaining individuals and
+corporations we need to get agreements
+from](https://docs.google.com/spreadsheets/d/18_0Hog_eSwES8lKwf7WJal3yBwwcYfvPu1yCfZnTcek/edit?usp=sharing),
+and reaching out to them, or letting us know at license-questions@llvm.org how
+we can reach out to them. Also, if you'd happen to have any other information that you think could help us, please do let us know at license-questions@llvm.org.
```

Comment:
linkify "license-questions@llvm.org"?

Reply:
In the review view on github this does not get linkified, but it does get linkified automatically by the hugo framework used to produce the blog post. Or in other words, in this review, you don't see it linkified, but it will be linkified on the blog.